mmif/serialize/mmif.py

```diff
@@ -607,3 +607,8 @@ def get_last(self) -> Optional[View]:
         for view in reversed(self._items.values()):
             if 'error' not in view.metadata and 'warning' not in view.metadata:
                 return view
+
+    def __delete_last(self, n: int) -> None:
+        list_reversed = list(reversed(self._items.keys()))[:n]
```

Comment:
Also, since this is an addition to the SDK core code, I'd love to have a unittest for this method. Please take a look at [`test_serialize`](https://github.com/clamsproject/mmif-python/blob/226ddf4086896c863c2d4260a9baa3d055c7434a/tests/test_serialize.py) and find a place to add one or two unittest for this deletion function, and write them.